packages/cli/src/commands/start.ts

```diff
@@ -115,20 +115,7 @@ export class Start extends BaseCommand {
 
 			await Container.get(InternalHooks).onN8nStop();
 
-			const skipWebhookDeregistration = config.getEnv(
-				'endpoints.skipWebhooksDeregistrationOnShutdown',
-			);
-
-			const removePromises = [];
-			if (!skipWebhookDeregistration) {
-				removePromises.push(this.activeWorkflowRunner.removeAll());
```

Comment:
we don't need to change it in this PR, but I think we should move `activeWorkflowRunner.removeAll` into a test utility, as it's only used in tests now.